conans/client/graph/graph_binaries.py

```diff
@@ -469,6 +469,9 @@ def _skip_binaries(graph):
         required_nodes.add(graph.root)
         for node in graph.nodes:
             if node.binary in (BINARY_BUILD, BINARY_EDITABLE_BUILD, BINARY_EDITABLE):
+                if node.skipped_build_requires:
```

Comment:
I think this does not cover the case of `tools.graph:skip_test=True` and you ask to build the recipe, right? We should also keep track of skipped tests

Reply:
I had that management too, but I removed it.
I think that is a valid use case. You can perfectly decide to build the package but skip building and running the tests, with ``tools.build:skip_test``, and for efficiency you can also decide to skip downloading the ``test_requires`` with this new conf. That means that if you are using ``tools.graph:skip_test`` you should probably use ``tools.build:skip_test`` too (unless you are using no test-requires, just building tests with ctest for example).